FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -255,7 +297,7 @@ if (( generate_template )); then
 	  mv ${LongDIR}/${TemplateID} ${LongDIR}/${TemplateID}.${TimeStamp}
 	fi
 
-	recon_all_cmd="recon-all.v6.hires"
+	recon_all_cmd="$queuing_command recon-all.v6.hires"
```

Comment:
If you have an `if` or `case` between the different submission types anyway, I wouldn't put the queueing tool in the "recon_all command" variable.